wordpress-mobile/WordPress-iOS
# [Share Extension] Fix textview inset

Fixes #12566 

With iOS 13 the Share Extension view hierarchy changed and the keyboard appears over the view.
This requires the text view inset to be calculated.

## To test:
- Run _WordPressShareExtension_ with this branch on iOS 13
- Share an item selecting WordPress
- Add long text in the text view and check you can scroll the whole content
- Run it on iOS 12 

Update release notes:
